src/modules/tasksClass.js

```diff
@@ -0,0 +1,88 @@
+let checkboxes;
+let descriptions;
+
+export default class Tasks {
+  tasks = [];
+
+  constructor() {
+    this.getFromLocalStorage();
+  }
+
+  setLocalStorage = (newTasks) => localStorage.setItem('tasks', JSON.stringify(newTasks));
+
+  getFromLocalStorage = () => {
+    if (localStorage.getItem('tasks')) this.tasks = JSON.parse(localStorage.getItem('tasks'));
+  };
+
+  updateTasks = () => {
+    this.getFromLocalStorage();
+    const temp = this.tasks;
+    document.querySelector('.mylist').innerHTML = '';
+    this.tasks.forEach((task, i) => {
+      task.index = i + 1;
+      document.querySelector('.mylist').innerHTML += `<li>
```

Comment:
The document.querySelector('.mylist') selector is used multiple times in the updateTasks() function. Instead, it could be declared as a variable at the beginning of the function and then reused, example: `const myList = document.querySelector('.mylist');`